src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -611,7 +634,7 @@ else if ("goog-remb".equals(rtcpFb.getAttribute("type")))
         }
 
         MediaStream mediaStream = getStream();
-        if (mediaStream != null && mediaStream instanceof VideoMediaStreamImpl)
+        if (mediaStream instanceof VideoMediaStreamImpl)
```

Comment:
Can mediaStream be null here, as you are getting rid of the check?

Reply:
A null check is not needed before using instanceof because the expression `x instanceof SomeClass` is `false` if x is `null`.